File: SMI-SDK/Java SDK/src/pt/sapo/gis/exception/ConfigurationException.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package pt.sapo.gis.exception;

/**
 * Class for Configuration exceptions on trace application
 * @author GIS Tam
 */
public class ConfigurationException extends TraceSDKException{

    public ConfigurationException() {
    }

    public ConfigurationException(String message) {
        super(message);
    }

    public ConfigurationException(Throwable cause) {
        super(cause);
    }

    public ConfigurationException(String message, Throwable cause) {
        super(message, cause);
    }    
}

File: SMI-SDK/.NET SDK/Library/gis/exception/ConfigurationException.cs
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
using System;
namespace pt.sapo.gis.exception
{

    /**
     * Class for Configuration exceptions on trace application
     * @author GIS Tam
     */
    public class ConfigurationException : TraceException
    {

        public ConfigurationException() {}

        public ConfigurationException(String message) : base(message) {}

        public ConfigurationException(String message, Exception cause) : base(message, cause) {}
    }
}